ml/rlvr/reward_verifier.py

```
@@ -0,0 +1,270 @@
+            Decimal(str(score[0])) * Decimal(str(score[1]))
+            for score in scores
+        )
+        confidence_sum = sum(Decimal(str(score[1])) for score in scores)
+
+        composite_reward = weighted_sum / confidence_sum if confidence_sum > 0 else Decimal("0.5")
+
+        return composite_reward
+
+    async def _save_reward_score(
+        self,
+        execution_id: uuid.UUID,
```

Ruff (0.14.5)

Unused method argument: `execution_id`

(ARG002)